index.html

```diff
@@ -428,7 +428,7 @@ <h2 class="white-header">FOSSASIA on social media</h2>
             <!-- Stop ignoring -->
             <!-- Start ignoring BootLintBear -->
                 <div class="col-md-3">
-                    <a class="twitter-timeline" data-width="370"
+                    <a class="twitter-timeline" data-width="370" data-height = "500"
```

Comment:
How does this code fix the issue you mentioned in the description?
Moreover, I can't find any such issue.
Can you elaborate this with the use of snapshots (esp. before and after, if possible)?